manual/manual/tutorials/coreexamples.etex

```diff
@@ -61,9 +63,9 @@ usual basic data types: booleans, characters, and immutable character strings.
 "Hello world";;
 \end{caml_example}
 
-Predefined data structures include tuples, arrays, and lists. General
-mechanisms for defining your own data structures are also provided.
-They will be covered in more details later; for now, we concentrate on lists.
+Predefined data structures include tuples, arrays, and lists. There are also
+general mechanisms for defining your own data structures.  These are records
+and classes, which will be covered in more detail later; for now, we concentrate on lists.
```

Comment:
_/me is not really a fan of even acknowledging classes exist this early..._

Reply:
I'll take out the reference to classes and leave records as an example.

While I can frequently identify when something is confusing or missing, it's of course challenging for me as a novice to come up with specific wording and accurate details.

Reply:
> I'll take out the reference to classes and leave records as an example.

I think citing variants at the same level of record would be nice, the two complements each other a lot and are fundamental parts of the type system.

> While I can frequently identify when something is confusing or missing, it's of course challenging for me as a novice to come up with specific changes to the wording and accurate details.

Sorry if my tone was too dry, I really appreciate your contribution and it will make for a nice improvemen … after a bit of editing.